### PR TITLE
Rebrand and webpack rework

### PR DESCRIPTION
**Pull request by Tj-Tracy**
First wave of rebrand work. I've edited the webpack to combine both the
certificate and base styles bundling, along with removal of errant files
that were being generated (we don't need js bundles, theres no js in
this repo).

  
image

### CODE REVIEW

File: edx_credentials_themes/static/edx.org/certificates/sass/_utilities.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+$professional-certificate-watermark-color: $xseries-color;
+
+$edx-e-color: #b72768;
+$edx-e-d-overlap-color: #78212e;
+$edx-d-color: #8a8c8f;
+$edx-d-x-overlap-color: #026ba4;
+$edx-x-color: #209fda;
+
+// #bootstrapupgrade
+// Colors from edX bootstrap. Remove these when we update the libraries.
```
Comment (crice100) [inline edx_credentials_themes/static/edx.org/certificates/sass/_utilities.scss:17]: A TODO might be helpful.

Do we have a ticket open so we don't forget to do this?

File: edx_credentials_themes/static/edx.org/certificates/sass/_print-rendering.scss [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    margin-top: 0 !important;
+
+    .accomplishment-stamp-platform {
+      display: inline-block;
+      vertical-align: top;
+      margin-bottom: 0 !important;
+      padding-top: 0 !important;
+      @include margin-right(25px);
+
+      // FIXME This is edx.org-specific and belongs in a themed stylesheet in credentials-themes
```
Comment (crice100) [inline edx_credentials_themes/static/edx.org/certificates/sass/_print-rendering.scss:113]: Did we mean to leave this in the PR?

File: edx_credentials_themes/static/edx.org/certificates/sass/_layouts-rendering.scss [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+}
+
+.accomplishment-stamps {
+  @extend .list-unstyled;
+  white-space: nowrap;
+
+  .accomplishment-stamp-platform {
+    margin-bottom: 15px;
+
+    // FIXME This is edx.org-specific and belongs in a themed stylesheet in credentials-themes
```
Comment (crice100) [inline edx_credentials_themes/static/edx.org/certificates/sass/_layouts-rendering.scss:160]: Same question.

File: edx_credentials_themes/static/edx.org/certificates/sass/_build.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// ------------------------------
+// MIT Professional Education: Shared Build
```
Comment (crice100) [inline edx_credentials_themes/static/edx.org/certificates/sass/_build.scss:2]: Is this file specific to a particular type of cert?

**Review by crice100 — COMMENTED** [review]
Could you also update the version?

https://github.com/edx/credentials-themes/blob/master/edx_credentials_themes/__init__.py

**Review by crice100 — APPROVED** [review]

File: edx_credentials_themes/static/edx.org/certificates/sass/_print-rendering.scss (hunk 2 above)
Comment (Tj-Tracy) [reply edx_credentials_themes/static/edx.org/certificates/sass/_print-rendering.scss:113]: This is an old comment. I've yet to go through these.

File: edx_credentials_themes/static/edx.org/certificates/sass/_layouts-rendering.scss (hunk 3 above)
Comment (Tj-Tracy) [reply edx_credentials_themes/static/edx.org/certificates/sass/_layouts-rendering.scss:160]: Anothero ld comment.

File: edx_credentials_themes/static/edx.org/certificates/sass/_build.scss (hunk 4 above)
Comment (Tj-Tracy) [reply edx_credentials_themes/static/edx.org/certificates/sass/_build.scss]: Deleted - Its for the base of the certificate themes. Not sure why it mentioned MIT here.

**Pull request merged**